pkg/install/generate.go

```diff
@@ -21,15 +21,17 @@ import (
 )
 
 const (
-	azureAuthDir       = "/.azure"
-	azureAuthFile      = azureAuthDir + "/osServicePrincipal.json"
-	gcpAuthDir         = "/.gcp"
-	gcpAuthFile        = gcpAuthDir + "/" + constants.GCPCredentialsName
-	openStackCloudsDir = "/etc/openstack"
-	openStackCADir     = "/etc/openstack-ca"
-	vsphereCloudsDir   = "/vsphere"
-	ovirtCloudsDir     = "/.ovirt"
-	ovirtCADir         = "/.ovirt-ca"
+	azureAuthDir             = "/.azure"
+	azureAuthFile            = azureAuthDir + "/osServicePrincipal.json"
+	gcpAuthDir               = "/.gcp"
+	gcpAuthFile              = gcpAuthDir + "/" + constants.GCPCredentialsName
+	awsDir                   = "/.aws"
+	awsBoundSASigningKeyFile = awsDir + "/" + constants.BoundServiceAccountSigningKeyFile
```

Comment:
needs a rename to be not restricted to aws

Reply:
done, thanks!